apps/consulting/pages/nebari-services.tsx

```diff
@@ -372,11 +372,8 @@ export const NebariServicesPage: FC<TContainerProps> = ({
     <section className="bg-violet">
       <h2 className="sr-only">Testimonials</h2>
       <div
-        className="p-4 mx-auto sm:px-[3.4rem] xl:px-[11.5rem] max-w-layout"
-        style={{
-          background:
-            'url(/nebari-services/quansight-logo-violet-grayscale.svg) right bottom / 325px no-repeat',
-        }}
+        // eslint-disable-next-line
```

Comment:
Had to disable eslint because it complained that bg-[url] and bg-right-bottom contradict each other, but they definitely do not